src/Components/CreateImageWizard/utilities/requestMapper.ts

```diff
@@ -587,7 +587,8 @@ const getCustomizations = (state: RootState, orgID: string): Customizations => {
     });
     files.push({
       path: '/usr/local/sbin/register-satellite',
-      data: btoa(encodeURIComponent(satCmd)),
+      data: btoa(satCmd),
```

Comment:
Are we not adding shebang then?

Reply:
No, it is not necessary, I have changed the service unit it now calls `bash -x /the/script`